app/models/requests/full_patron.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+module Requests
+  # FullPatron pulls all available data from both Alma and LDAP via Bibdata
+  class FullPatron
+    attr_reader :hash, :errors, :api_response, :uid
+    def initialize(uid:)
+      @errors = []
+      @uid = uid
+      @hash = patron_hash
+    end
+
+    private
+
```

Comment:
@maxkadel we're reviewing the PR with @sandbergja .
Can we move the `attr_reader :api_response, :uid` in the private?